radio/wifi/nm.go

```diff
@@ -56,6 +56,13 @@ const (
 	NmActiveConnectionStateDeactivated  NmActiveConnectionState = 4
 )
 
+type NmDevice struct {
+	nmPath      dbus.ObjectPath
```

Comment:
Because `type` and `interface` are reserved words and I couldn't think of better names